includes/Core/Authentication/Authentication.php

```diff
@@ -602,10 +602,10 @@ private function inline_js_setup_data( $data ) {
 
 		$data['isSiteKitConnected'] = $this->credentials->has();
 		$data['isResettable']       = $this->options->has( Credentials::OPTION );
-		$data['isAuthenticated']    = ! empty( $access_token );
+		$data['authenticated']      = ! empty( $access_token );
```

Comment:
I think changing this wasn't part of the ACs - maybe it was slightly vague, but only the API route should be modified, we don't need to update this legacy data here. Having this change here also makes this PR a lot bigger with more side effects.